release/air_tests/air_benchmarks/mlperf-train/compute_cpu_16.yaml

```diff
@@ -7,4 +7,9 @@ head_node_type:
     name: head_node
     instance_type: m5.4xlarge
 
-worker_node_types: []
+worker_node_types:
```

Comment:
Hm, is it possible that this could cause the other tests that use this same compute config to create workers? I'm not sure if they'll attempt to or not.

Reply:
Oh, if max_workers is 0, can we just remove these changes?

Reply:
Op, I see you're still adding things to this PR, will mark as author-action-required.

Reply:
Ah sorry yep! And yeah you're right, I updated recently to copy the config over!